tests/test_service.py

```diff
@@ -97,7 +138,7 @@ class TestValidateService(unittest.TestCase):
                "survey_id": "023",
                "tx_id": "0f534ffc-9442-414c-b39f-a756b4adc6cb",
                "version" : "0.0.1"
-            }'''
+            }''',
```

Comment:
is this needed?

Reply:
It just makes future PRs cleaner, as the line won't show up . as being changed.